2025-08-17 23:20:03,157 - ERROR - Error creating PDF test_output/5.3/en-US/Unreal_Engine_53_Documentation__Unreal_Engine_53_D.pdf: No wkhtmltopdf executable found: "b''"
If this file exists please check that this process can read it or you can pass path to it manually in method call, check README. Otherwise please install wkhtmltopdf - https://github.com/JazzCore/python-pdfkit/wiki/Installing-wkhtmltopdf
2025-08-17 23:20:07,859 - ERROR - Error creating PDF test_output/5.3/en-US/understanding-the-basics-of-unreal-engine/Just_a_moment.pdf: No wkhtmltopdf executable found: "b''"
If this file exists please check that this process can read it or you can pass path to it manually in method call, check README. Otherwise please install wkhtmltopdf - https://github.com/JazzCore/python-pdfkit/wiki/Installing-wkhtmltopdf
2025-08-17 23:20:11,322 - WARNING - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fd02d61ead0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/25b416b2-9a83-430a-87a5-e57de11a7e05
2025-08-17 23:20:11,324 - WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fd02d61e850>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/25b416b2-9a83-430a-87a5-e57de11a7e05
2025-08-17 23:20:11,326 - WARNING - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fd02d78fbb0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/25b416b2-9a83-430a-87a5-e57de11a7e05
2025-08-17 23:21:06,824 - ERROR - Relative URI reference without a base URI: <img src="images/static/document_list/empty_thumbnail.svg">
2025-08-17 23:21:06,826 - ERROR - Relative URI reference without a base URI: <img src="images/static/document_list/empty_thumbnail.svg">
2025-08-17 23:21:06,827 - ERROR - Relative URI reference without a base URI: <img src="images/static/document_list/empty_thumbnail.svg">
2025-08-17 23:21:18,033 - WARNING - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7ff4c276f750>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/15f39675-0046-4a34-a696-c451fff6cc71
2025-08-17 23:21:18,035 - WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7ff4c25ab4d0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/15f39675-0046-4a34-a696-c451fff6cc71
2025-08-17 23:21:18,037 - WARNING - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7ff4c25fe060>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/15f39675-0046-4a34-a696-c451fff6cc71
